src/main/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorWorkflowStep.groovy

```diff
@@ -67,11 +67,21 @@ class IqPolicyEvaluatorWorkflowStep
     this.jobCredentialsId = jobCredentialsId
   }
 
+
+  @DataBoundSetter
+  public void setIqApplication(final String iqApplication) {
+    this.iqApplication = new SelectedApplication(iqApplication)
+  }
+
+  /*@DataBoundSetter
+  public void setIqApplication(final ManualApplication iqApplication) {
+    this.iqApplication = iqApplication
+  }*/
+
   @DataBoundConstructor
-  IqPolicyEvaluatorWorkflowStep(final String iqStage,
-                                final String iqApplication) {
+  IqPolicyEvaluatorWorkflowStep(final String iqStage)
```

Comment:
Since iqApplication is required and it's always a type String, it can be added to DataBoundConstructor.

Reply:
> Since iqApplication is required and it's always a type String, it can be added to DataBoundConstructor.

No longer only a string